extensions/ql-vscode/src/remote-queries/view/CollapsibleItem.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import styled from 'styled-components';
+import { ChevronUpIcon, ChevronDownIcon } from '@primer/octicons-react';
+import { useState } from 'react';
+
+const Container = styled.div`
+  display: block;
+  vertical-align: middle;
+`;
+
+const TitleContainer = styled.span`
+  display: inline-block;
+`;
+
+const Button = styled.button`
+  display: inline-block;
+  background-color: transparent;
+  border: none;
+  padding-left: 0;
+  padding-right: 0.1em;
+`;
+
+const CollapsibleItem = ({
+  title,
+  children
+}: {
+  title: React.ReactNode;
+  children: React.ReactNode
+}) => {
+  const [isExpanded, setExpanded] = useState(false);
+  return (
+    <>
+      <Container>
+        <Button onClick={() => setExpanded(!isExpanded)}>
+          {isExpanded
+            ? <ChevronUpIcon size={16} />
+            : <ChevronDownIcon size={16} />
```

Comment:
Looking at the design, I think we want "ChevronDown" for the expanded list 🔽 
I can't see a specific design for the collapsed list, but probably "ChevronRight" ▶️ That seems fairly standard (e.g. in the Files Changed tab here on GH 😄)

Reply:
Thanks, I thought it looked a odd! 😅